fcm/fcm.go

```diff
@@ -99,22 +99,33 @@ func (f *FCM) ConnectToFcm(fcmNotifications chan<- notification.PrinterNotificat
 		go func() {
 			for {
 				raw_input, err1 := reader.ReadBytes('\n')
-				inputA := strings.SplitN(string(raw_input), "\\n", 2)
-				notification_size := inputA[0]
+				input_chunks := strings.SplitN(string(raw_input), "\n", 2)
+				notification_size := strings.TrimSpace(input_chunks[0])
 				notification_data1 := ""
-				if len(inputA) > 1 {
-					notification_data1 = inputA[1]
+				if len(input_chunks) > 1 {
+					notification_data1 = strings.TrimSpace(input_chunks[1])
```

Comment:
why trim space here? That may trim space that's embedded in the message body, right?

Reply:
Fixed.